### PR TITLE
blog(embertimes#129): add octane, december, holiday writeup

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

**Review by chancancode — APPROVED** [review]
Thank you, this looks great! 🎉

File: source/2019-12-20-the-ember-times-issue-129.md [hunk 1]
```diff
@@ -1,37 +1,33 @@
 ---
 title: The Ember Times - Issue No. 129
-author: Chris Ng, Amy Lam, Isaac Lee, the crowd
+author: Chris Ng, Amy Lam, Isaac Lee, Godfrey Chan, Jessica Jordan
```
Comment (chancancode) [inline source/2019-12-20-the-ember-times-issue-129.md:3]: Not sure if I deserve the credit here but sure! 😅

Comment (jayjayjpg) [reply source/2019-12-20-the-ember-times-issue-129.md:3]: Of course! :-)

File: source/2019-12-20-the-ember-times-issue-129.md [hunk 2]
```diff
@@ -1,37 +1,33 @@
...
+And on top of that, Ember Octane is **fully opt-in** and **interoperable** for existing Ember apps - which means you can try out the new way of building modern Ember Octane apps gradually, without having to rewrite your entire app today.
 
----
-
-## [Octane is Here 🎉](https://blog.emberjs.com/2019/12/20/octane-is-here.html)
-
-Ember [3.15](https://blog.emberjs.com/2019/12/20/ember-3-15-released.html) was released this week. In addition to the usual new features, improvements and bug fixes, this release also marks the official release of the Octane edition! Read more about what's new and what's changed in Octane in this summary by [Yehuda Katz (@wycats)](https://github.com/wycats).
+Want to know all about Ember Octane? Learn more about what's new in Octane and how to get started [in the official Ember Octane release announcement](https://blog.emberjs.com/2019/12/20/octane-is-here.html) by [Yehuda Katz (@wycats)](https://github.com/wycats)!
```
Comment (chancancode) [inline source/2019-12-20-the-ember-times-issue-129.md]:
```suggestion
Want to know all about Ember Octane? Learn more about what's new in Octane and how to get started [in the official Ember Octane release announcement](https://blog.emberjs.com/2019/12/20/octane-is-here.html) by [Yehuda Katz (@wycats)](https://github.com/wycats)! Need help? Check out **#topic-octane-migration** channel on the [Ember Discord](https://discordapp.com/invite/emberjs)!
```

**Pull request merged**